### PR TITLE
ws 30 get all species

### PR DESCRIPTION
**Pull request by BharathiRamachandran14**
We have updated the Repo, Service and Controller of Whale which enables us to get the whaleopedia.
We successfully tested the end points for the same.

image

### CODE REVIEW

File: backend/Repositories/WhaleRepo.cs [hunk 1]
```diff
@@ -12,5 +18,11 @@ public WhaleRepo(WhaleSpottingDbContext context)
         {
             _context = context;
         }
+         public IEnumerable<Species> GetAllSpecies()
+        {
+            return _context.Species
+                .Include(s => s.ConservationStatus)
+                .OrderByDescending(s => s.Name);
```
Comment (TimLeach635) [inline backend/Repositories/WhaleRepo.cs:25]: Why descending? Reverse alphabetical order seems like a strange choice, wouldn't ascending make more sense?

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
Only one change!

File: backend/Repositories/WhaleRepo.cs (hunk 1 above)
Comment (BharathiRamachandran14) [reply backend/Repositories/WhaleRepo.cs:25]: Agreed! We couldn't work out how to do it!

Comment (BharathiRamachandran14) [reply backend/Repositories/WhaleRepo.cs:25]: Please help!!!!

**Review by TimLeach635 — APPROVED** [review]

**Pull request merged**